backend/app/Http/Controllers/AnswerController.php

```diff
@@ -50,6 +51,25 @@ public function getAnswersByAttempt(Attempt $attempt)
         ], 201);
     }
 
+    public function getAnswersByUser(User $user) {
+
+        $attempts = $user->attempts;
+        $answerList = collect([]);
+
+        foreach ($attempts as $attempt) {
+            $answers = $attempt->answers;
+            foreach ($answers as $answer) {
+                $answer->word;
+            }
+            $answerList = $answerList->merge($answers);
+        }
```

Comment:
I think you can refine this, using `with()`. And it is a bad practice to use nested for loop. it is very expensive,  O(n^2),